jcommune-model/src/main/resources/org/jtalks/jcommune/migrations/V76__Add_column_Modification_Date and_User_Changed_to_PostComment_Table.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE `post_comment` ADD (
+  `MODIFICATION_DATE` datetime DEFAULT NULL,
+  `USER_CHANGED` BIGINT(20) DEFAULT NULL,
```

Comment:
since i made it in the same way as AUTHOR_ID - `AUTHOR_ID` bigint(20) NOT NULL,
if it isn't correct, say, pls, how it must be.